Sources/SwifterSwift/UIKit/UIImageExtensions.swift

```diff
@@ -309,6 +309,15 @@ public extension UIImage {
         self.init(cgImage: aCgImage)
     }
 
+    /// SwifterSwift: Create a new image from a base 64 string.
+    ///
+    /// - Parameters:
+    ///   - string: a base 64 string, representing the image
+    convenience init?(base64String string: String) {
+        guard let data = Data(base64Encoded: string, options: .ignoreUnknownCharacters) else { return nil }
```

Comment:
Why do you want to add `.ignoreUnknownCharacters`? It seems to me like this should fail if there are invalid characters included, no?

Reply:
I also was unsure about this.. Will remove it, thanks for the hint ^^